symengine/parser/parser.yy

```diff
@@ -180,6 +180,9 @@ expr:
 |
         '-' expr %prec UMINUS
         { $$ = neg($2); }
+|
+        '+' expr %prec UMINUS
```

Comment:
doesn't change the results, but maybe for consistency with the other rules? (along with adding a `%right UPLUS` below the existing `%right UMINUS` on line 89)
```suggestion
        '+' expr %prec UPLUS
```

Reply:
Ok I have fixed this